src/main/java/br/com/budgetboys/pf_finance_service/adapters/inbound/controller/income/IncomeController.java

```diff
@@ -0,0 +1,62 @@
+package br.com.budgetboys.pf_finance_service.adapters.inbound.controller.income;
+
+import br.com.budgetboys.pf_finance_service.adapters.outbound.service.income.IncomeService;
+import br.com.budgetboys.pf_finance_service.domain.income.IncomeCreateDTO;
+import br.com.budgetboys.pf_finance_service.domain.income.IncomeResponseDTO;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("income")
+public class IncomeController {
+
+    private final IncomeService incomeService;
+
+    public IncomeController(IncomeService incomeService) {
+        this.incomeService = incomeService;
+    }
+
+    @PostMapping("create")
+    public ResponseEntity<?> create(@RequestBody IncomeCreateDTO income) {
```

Comment:
Não é necessário fazer a validação do DTO no request do `create`?
```suggestion
    public ResponseEntity<?> create(@Valid @RequestBody IncomeCreateDTO income) {
```